File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int rem=0,or=x;
        long rev=0;
        if(x<0)
        {
            return false;
        }
        while(x>0){
        rem=x%10;
        rev=rem+(rev*10);
        x=x/10;
    }
    if(or==rev)
    {
        return true;
    }
    else
    {
        return false;
    }
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        rem = 0
        original = x
        rev = 0
        if x < 0:
            return False
        while x > 0:
            rem = x % 10
            rev = rem + (rev * 10)
            x = x // 10
        return original == rev